src/Components/AppShell/AppShell.js

```diff
@@ -95,14 +73,13 @@ class AppShell extends Component {
     const segment = nextProps.location.pathname.split('/')[1] || 'dashboard'
 
     // If we haven't initialized GA, do it.
-    if (!this.user) {
-      this._setupGoogleAnalytics()
+    if (!GA.isSetup) {
+      GA.setup(nextProps.settings.googleAnalyticsTracking, nextProps.settings.uuid)
+      GA.reportSettings(nextProps.settings)
     }
 
-    if (this.user) {
-      this.user.pageview(nextProps.location.pathname).send()
-      this.user.screenview(segment, 'Ganache', app.getVersion()).send()
-    }
+    GA.reportPageview(nextProps.location.pathname)
```

Comment:
would be better if these were actions being dispatched rather than direct calls to the GA lib.

Reply:
See my comment here: https://github.com/trufflesuite/ganache/pull/743/files/2a956a4bffd71db286859711a13816d915be2d55#r196992816